### PR TITLE
Adding skill node

### PR DESCRIPTION
**Pull request by kvmanohar22**

### CODE REVIEW

File: scripts/skills/skill_node.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+import rospy
+from krssg_ssl_msgs.msg import gr_Robot_Command
+from krssg_ssl_msgs.msg import gr_Commands
+
+pub = rospy.Publisher('/global_bot_data', gr_Commands, queue_size=1000)
```
Comment (annimesh2809) [inline scripts/skills/skill_node.py]: Let's change the rostopic name to `/bot_data` only.

File: scripts/skills/skill_node.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+rospy.init_node('bot_node', anonymous=True)
```
Comment (annimesh2809) [inline scripts/skills/skill_node.py]: Shouldn't the name be `skills_node` or something like that?

**Review by annimesh2809 — COMMENTED** [review]
:+1:

File: scripts/skills/skill_node.py (hunk 2 above)
Comment (kvmanohar22) [reply scripts/skills/skill_node.py]: Let me change them !

**Pull request merged**